scripts/masterCron.ts

```diff
@@ -8,15 +8,19 @@ import {
   getDefaultSigner,
 } from "./helpers/getDefaultEthersProvider";
 import moment from "moment";
-import deployments from "../constants/deployments.json";
+import deployments from "../constants/deployments-mainnet-cron.json";
```

Comment:
Is there a reason why we have to use `deployments-mainnet-cron.json` instead? We're generating `constants/deployments.json` using the `yarn export-deployments` command so we have a single source of truth